source/mir/lapack.d

```diff
@@ -18,18 +18,16 @@ static import lapack;
 
 public import lapack: lapackint;
 
-//enum for errors.
-private enum Error {
-    squareM = "The matrix must be square"
-};
-
 @trusted pure nothrow @nogc:
 
 /// `getri` work space query.
 size_t getri_wq(T)(Slice!(T*, 2, Canonical) a)
+in
+{
+    assert(a.length!0 == a.length!1, "getri: a must be a square matrix.");
```

Comment:
```
"getri: `a` must be a square matrix."
```

differentiate the parameter from the indefinite article. Probably could do with emphasis on the parameters in the other assert messages as well, but this one is more important.

Reply:
Is it ok to do it as 'a', or does it have to be `a`?

Reply:
Is for asserts, so it doesn't really matter.